### DISCUSSION TITLE
How to center Navbar items?

### DISCUSSION THREAD

**Discussion by ssinghm32** [Megabit/Blazorise#4101] [Q&A]
I'm using the Bar component to create a Bootstrap Navbar.
https://blazorise.com/docs/components/bar

How do I center the navbar items found in the <BarStart> tag?
The items start out left aligned, right next to the Nav Brand but I would like it centered in the navbar.

To take it a step further, I am looking for the navbar items to be centered on Desktop and left aligned in the collapseable mobile menu.

Thanks

**Comment by stsrki** [comment]
You can use TextAlignment

```razor
<BarItem>
    <BarLink To="" TextAlignment="TextAlignment.End">
        <BarIcon IconName="IconName.Dashboard" />
        Dashboard
    </BarLink>
</BarItem>
```

and

```razor
<BarDropdownItem To="tests/forms" TextAlignment="TextAlignment.End">
    Forms
</BarDropdownItem>
```

**Reply by ssinghm32** [reply]
@stsrki It looks like your solution only worked when the Navbar is rendered in the mobile view.  In desktop view, the navigation items are still left aligned.


I found what works better for what I wanted was adding the mx-auto class on the BarStart tag.  Not sure if this is the correct way of addressing it ??

```
<BarStart Class="mx-auto">
    <BarItem>
        <BarLink To="#home">Home</BarLink>
    </BarItem>
    <BarItem>
        <BarLink To="#docs">Documentation</BarLink>
    </BarItem>
    <BarItem>
        <BarDropdown>
            <BarDropdownToggle>Dropdown</BarDropdownToggle>
            <BarDropdownMenu>
                <BarDropdownItem>Action</BarDropdownItem>
                <BarDropdownDivider />
                <BarDropdownItem>Another action</BarDropdownItem>
            </BarDropdownMenu>
        </BarDropdown>
    </BarItem>
</BarStart>

```

**Reply by stsrki** [reply]
You can also use Blazorise utilities

```
<BarStart Margin="Margin.IsAuto.OnX">
```
